modules/pam_namespace/pam_namespace.c

```diff
@@ -1461,6 +1462,7 @@ static int create_polydir(struct polydir_s *polyptr,
     if (rc == -1) {
             pam_syslog(idata->pamh, LOG_ERR,
                        "Error creating directory %s: %m", dir);
+			freecon(oldcon_raw);
```

Comment:
Please wrap this with `#ifdef WITH_SELINUX`.

Reply:
Done